### PR TITLE
[Core] Create comparison list under Statistics tab

### PR DESCRIPTION
**Pull request by Pewtro**
Creates a comparison list under statistics that'll redirect to the player selection report (limited to this currently :( ) so that it makes for comparing easier for similiar kill times. 

image

image

### CODE REVIEW

File: src/interface/report/Results/EncounterStats.js [hunk 1]
```diff
@@ -213,6 +236,50 @@ class EncounterStats extends React.PureComponent {
...
+              <div>
+                {log.name} ({log.itemLevel})
+              </div>
+            </a>
+            <div>
+              {formatDuration(log.duration / 1000)} ({log.duration > this.props.duration ? ((log.duration - this.props.duration) / 1000).toFixed(1) + 's slower' : ((this.props.duration - log.duration) / 1000).toFixed(1) + 's faster'})
+            </div>
+          </div>
+          <div className="col-md-6">
+            {(log.total).toFixed(0)} DPS
```
Comment (MartijnHols) [inline src/interface/report/Results/EncounterStats.js]:
```suggestion
            {formatNumber(log.total)} DPS
```

**Comment by Pewtro** [comment]
That looked ugly imo: 
image

**Comment by Pewtro** [comment]
Using formatThousande looks 👌 
image

File: src/interface/report/Results/EncounterStats.js [hunk 2]
```diff
@@ -23,19 +23,24 @@ class EncounterStats extends React.PureComponent {
...
   };
 
-  LIMIT = 100;
+  LIMIT = 100; //Currently does nothing but if Kihra reimplements it'd be nice to have
   SHOW_TOP_ENTRYS = 6;
   SHOW_TOP_ENTRYS_AZERITE = 10;
+  SHOW_CLOSEST_KILL_TIME_LOGS = 10;
   metric = 'dps';
   amountOfParses = 0;
+  durationVariance = 0.2; //Marked in % to allow for similiar filtering on long/short fights
```
Comment (MartijnHols) [inline src/interface/report/Results/EncounterStats.js]:
```suggestion
  durationVariancePercentage = 0.2; //Marked in % to allow for similiar filtering on long/short fights
```

File: src/interface/report/Results/EncounterStats.js [hunk 3]
```diff
@@ -228,13 +295,13 @@ class EncounterStats extends React.PureComponent {
     this.amountOfParses = Object.values(this.state.mostUsedTalents[LEVEL_15_TALENT_ROW_INDEX]).reduce((total, parses) => total + parses);
     return (
       <>
-        <div className="panel-heading" style={{ padding: 20, marginBottom: '2em' }}>
-          <h2>Statistics of this fight of the top {this.amountOfParses} logs, ranked by {this.metric.toLocaleUpperCase()}</h2>
+        <div className="panel-heading" style={{ padding: 20, marginBottom: '2em', textAlign: 'center' }}>
```
Comment (MartijnHols) [inline src/interface/report/Results/EncounterStats.js]:
```suggestion
        <div className="panel-heading" style={{ padding: 20, marginBottom: '2em' }}>
```

**Pull request merged**